### ISSUE TITLE
Color contrast too low for... all schemes?

### ISSUE THREAD

**Issue by Artoria2e5** [shinigami-eyes/shinigami-eyes#62]
The green color used for trans-friendly is too bright, making it hard to read over a white background typical of... most web pages, something that can be verified both by feel and by a [WCAG 2.0 contrast ratio check](https://contrast-ratio.com/). That's... very inconvenient when you want to see your friend-to-be's name.

In general there is a bit of a contrast-issue with most of the colors used here. We get:

|       Color       	| vs. Black 	| vs. White 	|
|:-----------------:	|:---------:	|:---------:	|
| #77B91E           	| 8.74      	| 2.4       	|
| #991515           	| 2.47      	| 8.49      	|
| rgb(153, 0, 255)  	| 3.79      	| 5.52      	|
| rgb(179, 152, 0)  	| 7.4       	| 2.83      	|
| rgb(98, 161, 197) 	| 7.42      	| 2.82      	|
| #FB6A4A           	| 7.26      	| 2.89      	|
|rgb(56, 88, 152) (reference) | 3.01 | 6.95 |

(Note: the color contrast calculation is based on trichromat vision only.)

The only color that is close to looking okay for both backgrounds is the purple. 

* * *

### Solutions?

One idea would be to make every color act like the purple color shown above -- not too bright, not too dark. It would not completely meet the WCAG guideline of 4.5 over all backgrounds, but it's at least more usable.

The nicer approach would be using different color schemes based on testing whether the background is a dark color. It should be possible with a `class` injection similar to what is already done for the theme switching.

Tangentially, it would be nice to get an option to use arbitrary colors. They should show up if you just inject them into the `style` part of `body`, right?
